packages/documentation/src/stories/accessibility-practices/foundational-structure-and-semantics/reference-crossing-the-shadowdom/aria-describedby/examples/workingExample-IIa.sample.html

```diff
@@ -0,0 +1,16 @@
+<span id="descriptionId">My Description</span>
+
+<custom-component descriptionId="descriptionId">
+  ⏷ #shadow-root (open)
+    <div ref="targetRef" role="button" tabindex="0" aria-describedby>
+      <!--The aria-describedby attribute is set automatically by the browser -->
+        <custom-icon />
```

Comment:
Wrong indentation.
The `<custom-icon />` tag should be at the same level as the comment above.

Reply:
I think you missed that one, didn't you?